### DISCUSSION TITLE
Feature Request: Strikethrough in web edit toolbar

### DISCUSSION THREAD

**Discussion by renatolond** [community/community#31461] [Other Feature Feedback, Questions, & Ideas]
Hi!

I would really like to see a strikethrough button on the toolbar (and an associated shortcut) the same way there's one for bold and italic. My use case is in PRs or issues when we have parts of the text that are no longer valid (for instance, because the PR was a draft and the draft part was addressed since). There's a way to do it manually, but it would be very nice to have a shortcut/button and avoid having to do it by hand.

It seems this already exists in the mobile version, according to https://github.com/orgs/community/discussions/16.

**Comment by jfullstackdev** [comment]
```
~~strikethrough~~
```
rendered

~~strikethrough~~

that is strike through, it's a basic Markdown syntax

**Reply by renatolond** [reply]
Indeed, but what I want is a shortcut that adds it automatically the same way pressing `ctrl+i` over a selected text does. Also, a button that adds strikethrough in the bar (by the B and the _I_)
